modules/series-service-impl/src/main/java/org/opencastproject/series/impl/persistence/SeriesEntity.java

```diff
@@ -49,17 +53,39 @@
  * rules.
  *
  */
-@Entity(name = "SeriesEntity") @IdClass(SeriesEntityId.class)
+@Entity(name = "SeriesEntity")
+@IdClass(SeriesEntityId.class)
 @Access(AccessType.FIELD)
-@Table(name = "oc_series")
+@Table(name = "oc_series",
+    indexes = {
+        @Index(name = "IX_oc_series_modified_date", columnList = ("modified_date")),
+    }
+)
 @NamedQueries({
-    @NamedQuery(name = "Series.findAll", query = "select s from SeriesEntity s"),
-    @NamedQuery(name = "Series.getCount", query = "select COUNT(s) from SeriesEntity s"),
+    @NamedQuery(
+        name = "Series.findAll",
+        query = "select s from SeriesEntity s where s.deletionDate is null"
+    ),
+    @NamedQuery(
+        name = "Series.getCount",
+        query = "select COUNT(s) from SeriesEntity s where s.deletionDate is null"
+    ),
     @NamedQuery(
         name = "seriesById",
         query = "select s from SeriesEntity as s where s.seriesId=:seriesId and s.organization=:organization"
     ),
-    @NamedQuery(name = "allSeriesInOrg", query = "select s from SeriesEntity as s where s.organization=:organization")
+    @NamedQuery(
+        name = "Series.getAllModifiedSince",
+        query = "select s from SeriesEntity as s "
+            + "where s.modifiedDate >= :since "
+            + "order by s.modifiedDate asc"
+    ),
+    @NamedQuery(
+        name = "Series.getAllModifiedInRange",
+        query = "select s from SeriesEntity as s "
+            + "where s.modifiedDate >= :from and s.modifiedDate < :to "
```

Comment:
`from` alone sounds weird to me. And the pair `since` and `to` also sound weird. Which is why I opted for these names. But I don't really care. If you want, I can change `since` to `from`, for example.